Extensions/WikiUpdater/WikiUpdaterTask/src/GitWikiFuntions.ts

```diff
@@ -90,7 +99,7 @@ export async function UpdateGitWikiFile(repo, localpath, user, password, name, e
         }
         logInfo(`Cleaned ${localpath}`);
 
-        await git.silent(true).clone(remote, localpath);
+        await git.silent(true).clone(remote, localpath, extraHeaders);
```

Comment:
This pull and push commands will not work without the header as well. Adding it to local git config would fix that.
```suggestion
        await git.silent(true).clone(remote, localpath, extraHeaders);
        if (injectExtraHeader) {
            await git.addConfig("http.extraheader", "AUTHORIZATION: bearer ${password}");
        }
```

Reply:
Actually, this shoud be done after the git.cwd command but I was not able to put a comment in there.